concepticondata/conceptlists/Gauchat-1925-480.tsv

```diff
@@ -68,29 +68,29 @@ Gauchat-1925-480-66	66	is narrow	est étroite	strĭcta	22	1267	NARROW
 Gauchat-1925-480-67	67	The attic	Le grenier	granariu	24		
 Gauchat-1925-480-68	68	and the «beater»	et le „battoir“	batt + atōriu	24		
 Gauchat-1925-480-69	69	are very close	sont tout près	prĕssu	24		
-Gauchat-1925-480-70	70	of the village	du village	villaticu	24		
+Gauchat-1925-480-70	70	of the village	du village	villaticu	24	930	VILLAGE
 Gauchat-1925-480-71	71	Let's go	Allons-y	ambulemus	24		
 Gauchat-1925-480-72	72	together	ensemble	insĭmul	24	2482	TOGETHER
 Gauchat-1925-480-73	73	Be careful	Prends garde	cavica	26		
 Gauchat-1925-480-74	74	to stumble	de t‘“encoubler“	incopulare	26		
 Gauchat-1925-480-75	75	The bridge	Le pont	pŏnte	26	1011	BRIDGE
-Gauchat-1925-480-76	76	is	est en	est ĭn	26		
+Gauchat-1925-480-76	76	is of	est en	est ĭn	26		
 Gauchat-1925-480-77	77	stone	pierre	pĕtra	26	857	STONE
 Gauchat-1925-480-78	78	of iron	en fer	fĕrru	26		
 Gauchat-1925-480-79	79	It is new	Il est neuf	nŏvu	28		
-Gauchat-1925-480-80	80	broad	large	largu	28	2531	BROAD
+Gauchat-1925-480-80	80	wide	large	largu	28	1243	WIDE
 Gauchat-1925-480-81	81	The wall	Le mur	mūru	28	933	WALL (OF HOUSE)
-Gauchat-1925-480-82	82	has started	est commencé	cuminitiatu	28		
+Gauchat-1925-480-82	82	is begun	est commencé	cuminitiatu	28		
 Gauchat-1925-480-83	83	The journey	La tour	tŭrre	28	2393	JOURNEY
-Gauchat-1925-480-84	84	(has) started	(est) commencée	cuminitiata	28		
+Gauchat-1925-480-84	84	(is) started	(est) commencée	cuminitiata	28		
```

Comment:
dito, (has), sorry, otherwise this sounds very weird, and one doesn't understand it.